va/va.h

```diff
@@ -4224,9 +4225,15 @@ typedef struct _VASurfaceDecodeMBErrors {
 /**
  * After the application gets VA_STATUS_ERROR_DECODING_ERROR after calling vaSyncSurface(),
  * it can call vaQuerySurfaceError to find out further details on the particular error.
- * VA_STATUS_ERROR_DECODING_ERROR should be passed in as "error_status",
- * upon the return, error_info will point to an array of _VASurfaceDecodeMBErrors structure,
- * which is allocated and filled by libVA with detailed information on the missing or error macroblocks.
+ * VA_STATUS_ERROR_DECODING_ERROR should be passed in as "error_status".
+ *
+ * After the applications get VA_STATUS_HW_BUSY or VA_STATUS_SUCCESSFULL from vaSyncSurface(),
+ * it still can call vaQuerySurfaceError to find out further details to know if has real hw reset
+ * happened on this surface since umd and kmd could recover the context from reset with success in sometimes.
+ * VA_STATUS_HW_BUSY or VA_STATUS_SUCCESSFULL also could be passed in as "error_status".
```

Comment:
if there are one reset, and after re-run a while, there are another reset. how to know it?
each frame I need call it, first frame it is VADecodeReset, next frames it is invalid, then VADecodeReset again

Reply:
If user wants to know all the reset frames, then user needs to call vaQuerySurfaceError() for every frame and recode the reset information. 
If user don't care about the reset, he can just call vaQuerySurfaceError() when get error from syncSurface().